test/quantization/fx/test_quantize_fx.py

```diff
@@ -4035,6 +4035,9 @@ def forward(self, x):
             (True, False),  # functional relu
         )
         for dim, quant_type, use_bias, has_relu, f_relu in options:
+            # TODO(andrew): fix bias case
```

Comment:
are we going to support bias in the same PR? or is it already supported?

Reply:
looks like it is already handled

Reply:
It currently fails the test with "tensor values not close enough". I am investigating and will push a fix within this PR.

Reply:
why? aren't we producing the same model in the end?

Reply:
or are you comparing reference with non-reference again? which we didn't do before?

Reply:
It was because both models with and without bias matched the same pattern, but the lowering code assumed the model doesn't have a bias in both cases. This is fixed in the latest commit which matches the patterns manually and makes no such assumptions